factory/libgemini/gem_settings.py

```diff
@@ -86,3 +86,17 @@ class GemSettings(structy.Struct):
 
     lfo_2_factor: structy.Fix16 = 0
     """LFO 2's factor."""
+
+    castor_quantize: bool = False
+    """Quantize pitch CV inputs. This only affects the CV input, not the
+            pitch knob, so that the oscillator can still be tuned. The base CV
+            offset is added before quantization, so that the quantizer can be
+            calibrated against an external CV source.
+
+            If you find that Castor & Pollux's pitch keeps jumping back and forth
```

Comment:
We could add some hysteresis to avoid this. We don't necessarily have to do it in this PR, but might be a good follow-up issue.

Reply:
Hmm - I'm not sure that's a full solution. Imagine the user is using a keyboard which happens to be misaligned so that the voltages are right near our note borders. Then adding hystersis will mean each individual note has a stable pitch, but the result of each key will still be unpredictable.

So we certainly could add some hystersis, but I think we'd still need something to answer "Help, the quantizer is doing inconsistent things!"

Reply:
Although! Maybe that failure mode would be easier to describe, because it's either:

Without hysteresis => the pitch could either flap slowly (sounds like changing notes) or quickly (which will do horrible FM things to the audio).

With hysteresis => Keys produce definite, but inconsistent, notes.

Reply:
Exactly, that's my thinking as well. Again, we don't have to do it in this change. :)